packages/stripe-integration/.eslintrc.json

```diff
@@ -6,7 +6,8 @@
       "rules": {
         "@typescript-eslint/no-unsafe-argument": "off",
         "@typescript-eslint/no-unsafe-assignment": "off",
-        "@typescript-eslint/no-unsafe-member-access": "off"
+        "@typescript-eslint/no-unsafe-member-access": "off",
+        "@typescript-eslint/consistent-type-assertions": "off"
```

Comment:
if we need this rule only for test, will be better to keep this rule only for test files
```
{
      "files": ["*.test.tsx"],
      "rules": {
          "@typescript-eslint/consistent-type-assertions": "off"
        }
}
```